backend/src/api.ts

```diff
@@ -238,6 +239,11 @@ loginCheckedPost('/member-archive', async (req, user) => ({
   archive: await generateMemberArchive(req.body, user, req.query.semesters as number | undefined)
 }));
 
+loginCheckedGet('/member/:email/:category', async (req, user) => {
+  const result = await checkMemberMeetsCategory(req.params.email, user, req.params.category);
+  return { result };
```

Comment:
Right now if `result` returns `undefined` the response just looks like:

```
{}
```
How about we make the response a little more clear for the uncertain cases? Maybe "pass" | "fail" | "no data"? Feel free to adjust naming as you see fit.